website/blog/modules/ROOT/pages/13-mill-build-tool-v1-0-0.adoc

```diff
@@ -0,0 +1,366 @@
+- `import $ivy` for configuring dependencies for your `build.mill`
+- `import $repo` for configuring maven repositories used to resolve dependencies for compiling your `build.mill`
+
+These are all configuration values that need to be used early on in the Mill
+bootstrapping process, and thus we couldn't rely on them being configured in the "main"
+`build.mill` config-as-code. For example, before even compiling your `build.mill`, Mill
+already needs to know what version of Mill you want to use and what JVM to run it on!
+
+With Mill's YAML build headers, we can consolidate this zoo of different configuration
+styles into a single compact block at the top of every `build.mill`. While the older
+configuration styles continue to be supported for migration-compatibility, using
+Mill's build headers is the recommended approach for configuring these values going forward.
+
+Lastly, build headers are expected to future-proof this "early" configuration
+and allow all sorts of interesting use cases in future. For example, we can extend this
+format to support running self-contained Java/Scala/Kotlin scripts that contain both their
+dependency configuration and code, similar to
+https://docs.astral.sh/uv/guides/scripts/#running-a-script-without-dependencies[uv scripts]
+in Python.
+
+=== Mill Support for Kotlin Builds is now Stable
+
+Mill 1.0.0 includes substantial improvements for building Kotlin projects with Mill:
+
+* {link-pr}/4557[#4557], {link-pr}/4786[#4786], {link-pr}/4771[#4771],
+{link-pr}/4779[#4779], {link-pr}/4797[#4797], {link-pr}/4963[#4963]
+
+These PRs really flesh out the previously-experimental support for Kotlin projects in Mill:
+main class discovery, compiler plugins, BuildInfo support, etc.. Other PRs upstream in the
+https://get-coursier.io/[Coursier] add support for resolving Kotlin Multiplatform dependencies,
+supporting Kotlin-JS and Kotlin-Android projects.
+
+With 1.0.0, Kotlin support in Mill is no longer experimental, and we are enabling
+binary-compatibility enforcement for `mill.kotlinlib` just as we already have for
+`mill.javalib` and `mill.scalalib`. We hope that you will try out Mill in your Kotlin
+projects, and let us know how it goes in https://github.com/com-lihaoyi/mill/discussions[Mill's Github Discussions].
+
+
+=== Mill Support for Android Builds
+
+A huge amount of work went into 1.0.0 improving Mill's support for Android builds. Android
+apps have traditionally only been buildable using Gradle, and Mill is one of the only
+other build tools that you can use as an alternative. While
+in 0.12.x Android support was a demo-quality integration, in 1.0.0 it has been
+fleshed out into a robust and complete framework that can build many of the
+sample apps available in the Android ecosystem.
+
+For example, Mill is now able to build, run, and test the https://github.com/android/compose-samples/tree/main/JetLagged[JetLagged Android Example App]:
```

Comment:
perhaps the todo example will be more impressive, jetlagged has a nicer UI, but the todo app has hilt + the android tests run the whole app flow while the jetlagged only comes with an app is alive test